src/core/xref.js

```diff
@@ -156,60 +176,62 @@ function getRequestEntry(elem) {
     const inlineRefs = [...bibrefs].map(el => el.textContent.toLowerCase());
     const uniqueInlineRefs = [...new Set(inlineRefs)].sort();
     if (uniqueInlineRefs.length) {
-      specs.unshift(uniqueInlineRefs);
+      specs.push(uniqueInlineRefs);
```

Comment:
Does this create a behavior difference?

Reply:
No. Earlier, we were doing this the end of all processing, and then unshifting. Now, we push early. The priority in fallback chain remains same,